crates/runtime/src/dataconnector/debug.rs

```diff
@@ -19,7 +17,7 @@ pub struct DebugSource {}
 
 impl DataConnector for DebugSource {
     fn new(
-        _auth_provider: AuthProvider,
+        _secret: super::Secret,
```

Comment:
```suggestion
        _secret: Secret,
```

Add `use crate::secrets::Secret` to the top of the file